requirements.txt

```diff
@@ -20,18 +18,13 @@ PyYAML
 # flake8 is a superset of pycodestyle
 flake8
 
-# 2.6.7 uses invalid Python 2 syntax
-GC3Pie!=2.6.7; python_version < '3.0'
-GC3Pie; python_version >= '3.0' and python_version < '3.11'
+GC3Pie; python_version < '3.11'
 python-graph-dot
 python-hglib
 requests
 
 archspec
 
-# cryptography 3.4.0 no longer supports Python 2.7
-cryptography==3.3.2; python_version == '2.7'
 cryptography; python_version >= '3.5' and python_version < '3.11'
 
-# rich is only supported for Python 3.6+
 rich; python_version >= '3.6'
```

Comment:
strip off the `; python_version >= '3.6'` ?

Reply:
f9309b8